README.md

```diff
@@ -117,6 +118,16 @@ If the hook you're looking for hasn't been created yet, see the [develop a hook]
 
 Once you've added a hook as a dependency, it can be registered at the global, client, or flag invocation level.
 
+### Tracking
+
+The [tracking API](https://openfeature.dev/specification/sections/tracking/) allows you to use 
+OpenFeature abstractions to associate user actions with feature flag evaluations.
+This is essential for robust experimentation powered by feature flags. Note that, unlike methods 
+that handle feature flag evaluations, calling `track(...)` may throw an `IllegalArgumentException` 
+if an empty string is passed as the `trackingEventName`.
+
```

Comment:
Could you please show a quick example of how to call the track method?

Reply:
done 👍